test/controllers/returns/ImportedPlasticPackagingControllerSpec.scala

```diff
@@ -158,5 +178,23 @@ class ImportedPlasticPackagingControllerSpec extends SpecBase with MockitoSugar
         redirectLocation(result).value mustEqual controllers.routes.JourneyRecoveryController.onPageLoad.url
       }
     }
+
+    "submit must redirect to the mini-cya page if the answer has not changed" in {
+
+      when(mockUserAnswers.change(any(), any())(any())).thenReturn(None) // Respond saying the user-answer hasn't changed
+      when(mockUserAnswers.get(eqq(ImportedPlasticPackagingPage))(any())).thenReturn(Some(true)) // TODO shouldn't need this?
+
+      // TODO all these running() unit tests should go...
```

Comment:
Again lots of TODOs. Do we have tickets to ensure we mop this up and don't leave the code littered with TODOs?

Reply:
This TODO though... I do think we should be doing something about these unit tests sooner than later... @liampan may feel the same?